src/users/entities/user.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { UserRole } from "src/util/role.enum";
+import { Exclude } from "class-transformer";
+
+@Entity()
+export class User {
+    @PrimaryGeneratedColumn()
+    id: number
+
+    @Column()
+    fullName: string
+
+    @Column({unique: true})
+    email: string
+
+    @Column()
+    @Exclude()
+    password: string
+
+    @Column()
+    roles: UserRole
```

Comment:
make sure the timestamp `created_at` and `updated` are automatically added to the database.

If they are currently not available please add them